### PR TITLE
Remove binder/start

### PR DESCRIPTION
**Pull request by jhamman**
asking for a review from @mrocklin and @rsignell-usgs.

### CODE REVIEW

**Comment by mrocklin** [comment]
What's here seems fine to me.  Were there changes to `start` that you wanted to make here?

**Comment by jhamman** [comment]
@mrocklin - thanks for catching that. I had messed up a merge. Things are square now.

File: {{cookiecutter.project_slug}}/binder/start [hunk 1]
```diff
@@ -1,21 +1,9 @@
 #!/bin/bash
 
-export DASK_KUBERNETES__WORKER_TEMPLATE_PATH=${PWD}/.dask/config.yaml
-export DASK_KUBERNETES__WORKER_NAME=dask-{JUPYTERHUB_USER}-{uuid}
-
-# Replace DASK_DASHBOARD_URL with the proxy location
-sed -i -e "s|DASK_DASHBOARD_URL|/user/${JUPYTERHUB_USER}/proxy/8787|g" binder/jupyterlab-workspace.json
```
Comment (mrocklin) [inline {{cookiecutter.project_slug}}/binder/start]: I think that we might still need this?

Comment (mrocklin) [reply {{cookiecutter.project_slug}}/binder/start]: Just the bottom line

**Comment by mrocklin** [comment]
Looks good to me

**Pull request merged**